src/render.js

```diff
@@ -55,7 +55,17 @@ module.exports = function render() {
 
   if (store._emitSelectionChange) {
```

Comment:
When deselecting a point by clicking on the feature in `direct_select` mode the `draw.selectionchange` does not emit. This action keeps the user in direct_select. If I click off of the feature (which moves me to `simple_select`) the event is emitted.

It should be emitted both times.